[3.1] Scene Cache & Curve Editor (#1588)
cocos-creator/creator-docs: en/editor/preferences/index.md

3. We welcome users to try out and give us your feed backs via our [forum](https://discuss.cocos2d-x.org/c/33), you can make those features more powerful and easier to use in your own projects.

![laboratory](./index/laboratory.png)

> -**Turn on baking function**: Used to turn on the baking function, for details, please refer to [Light Map](./../../concepts/scene/light/lightmap.md)。

Reviewer: ```suggestion
- **Turn on baking functions**: Used to turn on the baking functions, for details, please refer to the [Light Map](./../../concepts/scene/light/lightmap.md) documentation.
```

> -**Scene Instant Cache**: Used to turn on the scene instant cache function, which is enabled by default. After opening, the scene file will be cached to the `temp/scene/[SCENE_UUID]/[TIME].json` location in the project directory at intervals during the scene editing process. In case of emergencies such as scene crash, progress Stuck, etc. Next time you open the editor, a pop-up window will ask whether to apply the latest scene file in the cache. It should be noted that once the scene is opened normally, the cache files before opening will be cleaned up. If you have special needs and need to browse the specified cached scene files by yourself, please operate with the corresponding scene closed.

Reviewer: ```suggestion
- **Scene Instant Cache**: Used to turn on the scene instant cache function, which is enabled by default. After opening, the scene file will be cached to the `temp/scene/[SCENE_UUID]/[TIME].json` location in the project directory at intervals during the scene editing process. In case of emergencies such as scene crash, progress stuck, etc. Next time the editor is opened, a pop-up window will ask whether to apply the latest scene file in the cache. It should be noted that once the scene is opened normally, the cache files before opening will be cleaned up. For special needs and the need to browse the specified cached scene files manually, please operate with the corresponding scene closed.
```
